src/api/holding/content-types/holding/lifecycles.js

```diff
@@ -0,0 +1,24 @@
+const fetch = require("node-fetch");
+
+module.exports = {
+    async beforeUpdate(event) {
```

Comment:
Maybe this does not include to execute the fetching when creating a new resource, only when updating.